test/ts-parser.test.js

```diff
@@ -31,10 +50,12 @@ describe('TypeScript Parser Test', function() {
   });
 
   it('should parse Array.includes with es2016', function() {
-    this.timeout(90000);
     var file = path.join(__dirname, 'fixtures/ts/Greeter.es2016.ts');
     var tsFiles = [file];
-    var tsParser = new TSParser(tsFiles, {target: 'es2016'});
+    var tsParser = new TSParser(tsFiles, {
+      target: 'es2016',
```

Comment:
IIRC, ES6=ES2015, ES2016 added only few features, the next big revision is ES2017 with async/await. Did you perhaps mean `es2015` as the taget?

I am surprised the target actually matters.

Anyhow, I don't have much opinions about this, just found a suspicious string. Please use whatever value makes most sense to you and works well.